packages/vats/src/bridge.js

```diff
@@ -15,76 +14,144 @@ import { makeWithQueue } from '@agoric/internal/src/queue.js';
  * @typedef {object} BridgeDevice
  * @property {(dstID: string, obj: any) => any} callOutbound
  * @property {(handler: { inbound: (srcID: string, obj: any) => void}) => void} registerInboundHandler
+ * @property {() => void} unregisterInboundHandler
  */
 
 /**
- * Create a handler that demuxes/muxes the bridge device by its first argument.
- *
- * @param {typeof import('@endo/far').E} E The eventual sender
+ * @param {import('@agoric/zone').Zone} zone
+ */
+export const zoneScopedManager = zone => {
+  const makeScopedManager = zone.exoClass(
+    'BridgeScopedManager',
```

Comment:
While I agree that the bridgeManager/scopedManager descriptions could be clarified and refactored, I consider that out of scope for this PR.  I'm not trying to improve the implementation, I'm focused on making it upgradable so that we have the option of improving it in the future.

Reply:
> @michaelfig given the new Bridge Channel terminology, would you mind updating the cosmic-swingset and golang layers too where applicable?

In the current PR, the terminology change is contained in the bridge vat.  Cosmic-swingset and golang don't have a notion of "bridge manager", much less "bridge channel".  However, propagating the terminology change to the naming of arguments and internal bindings in other vats and contracts is ~150 changes to ~20 files.

That work properly belongs in my "someday" pile and not in this PR.  Do you think that the rotation of BridgeManager/ScopedManager to BridgeManager/BridgeChannel in `bridge.js` is worthwhile (and maybe the global renaming of the ScopedManager type as well), or should I drop that commit from this PR and add it to the "someday" pile?

Reply:
Yeah I figured it might be too onerous to change today. I suspect that such a rename would have impact on some of the durability logic, so I'm ok to keep the rename limited to the vat side of things for now.

Reply:
Maybe bad form to reply to my own message, but I've made up my mind (I think).  I'll stash the terminology change commit and remove it from this PR in the interest of keeping this PR more focused.  I'll resurrect that stash in the form of a comprehensive PR if there's a reason to do so in the future.